app/helpers/schools/dashboard_helper.rb

```diff
@@ -2,8 +2,24 @@
 
 module Schools
   module DashboardHelper
-    def manage_ects_and_mentors?(school_cohort)
-      school_cohort.full_induction_programme? || school_cohort.core_induction_programme?
+    def actions?(participants)
+      participants.orphan_ects.any?
+    end
+
+    def ect_count(school_cohorts)
+      school_cohorts.sum { |sc| sc.current_induction_records.ects.count }
```

Comment:
```suggestion
      school_cohorts.sum { |sc| sc.current_induction_records.count }
```

The induction records have one participant, so I think counting the `current_induction_records` should be enough.

Also, Not sure if displaying the ects count is hugely important, so I wonder if it worth to add some specs to ensure the method is counting the `current_induction_records`. Someone could easily be confused and change the method to sum the `induction_records` or `active_induction_records`, which would count more induction records than intended.

Reply:
This scope (.ects) is to pick only current induction records belonging to an ECT.